messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/ConnectionFactoryAttributes.java

```diff
@@ -464,6 +489,8 @@ interface Common {
                 create(SCHEDULED_THREAD_POOL_MAX_SIZE, "scheduledThreadPoolMaxSize", true),
                 create(THREAD_POOL_MAX_SIZE, "threadPoolMaxSize", true),
                 create(GROUP_ID, "groupID", true),
+                create(DESERIALIZATION_ALLOWLIST, "deserializationWhiteList", true),
+                create(DESERIALIZATION_BLOCKLIST, "deserializationBlackList", true),
```

Comment:
I believe the string constants here are incorrect.

Reply:
No, that's the resource adapter property which hasn't changed since it is an Artemis class